shared/config.json

```diff
@@ -0,0 +1,81 @@
+{
+    "geocoding": true,
+    "census_data": true,
+    "_backend": "Configuration used by the backend.",
```

Comment:
?

Reply:
this is weird? why not just nest it?

frontend: {
blah
},
backend : {
blah
}

Reply:
Open to suggestions, just wanted to import the same thing for now